Content.Server/Fluids/EntitySystems/PuddleSystem.Evaporation.cs

```diff
@@ -55,6 +56,10 @@ private void TickEvaporation()
                 Spawn("PuddleSparkle", xformQuery.GetComponent(uid).Coordinates);
                 QueueDel(uid);
             }
+
+            var ev = new SolutionContainerChangedEvent(puddleSolution, puddle.SolutionName);
+            RaiseLocalEvent(uid, ref ev);
+            Dirty(puddle.Solution.Value.Owner, puddle.Solution.Value.Comp);
```

Comment:
This should just call solutioncontainersystem instead which handles the events rather than manually doing solution handling.